homeassistant/components/ozw/websocket_api.py

```diff
@@ -106,6 +108,59 @@ def _call_util_function(hass, connection, msg, send_result, function, *args):
     connection.send_result(msg[ID])
 
 
+def _get_config_params(node, *args):
+    raw_values = get_config_parameters(node)
+    config_params = []
+
+    for param in raw_values:
+        schema = {}
```

Comment:
How will this be converted by voluptuous_serialize? In other places we default to a raw list that is not converted.

https://github.com/home-assistant/core/blob/8895c65dd497c3630dd193dc95c4ce0b4a45d960/homeassistant/components/config/config_entries.py#L50-L55

Reply:
If `schema` is empty voluptuous_serialize just converts it to `[]`

Reply:
Ok.